.changeset/thirty-ads-wonder.md

```diff
@@ -0,0 +1,5 @@
+---
+'@api3/airnode-validator': minor
```

Comment:
Maybe this should be patch? This is more fixing a bug isn't it?

```suggestion
'@api3/airnode-validator': patch
```

Reply:
Yeah, patch is better.